### ISSUE TITLE
movestone strange movements

### ISSUE THREAD

**Issue by elettrico** [mt-mods/digistuff#50]
Hi,
I have three vertical digilines with a movestone in each of them, all connected to the same channel.
When i send the absmove or relmove commands to the channel, specifying only and Y position, I expect every movestone to move vertically along his path, instead some of them start moving horizontally through the digilines, reach the leftmos vertical digiline and then start moving vertically. The horizontal movement not only happen when the digilines are next to each other, but also when they are separated. Using relmove make the movement stranger, having one movestone move correctly, while the other move along the Y axis but exponentially (like, I'll pass Y=3: the first move 3 blocks, the second 9, and so on). This behavior can be observed also when two movestones are placed on the same digiline.
I don't think this the expected behaviour. It sound to me like somewhat the coordinates are shared between the movestones connected to the same channel, but they should be isolated for every movestone, because a digiline can send command to multiple movestones (in my case, for example, I want to do a piece of floor moving).
(A temporary workaround is to use a different channel for every movestone).

**Comment by OgelGames** [comment]
Yeah, movestones are still unstable, that's why they don't have a craft recipe. I think they need to be rewritten to ever work properly.

> Also, I've noticed it's impossible to use negative relmove, I mean, Y=-3 really does nothing.

That should be fixed by #43, make sure you're using the latest version. (not the one currently on ContentDB)

**Comment by wsor4035** [comment]
a user has reported that this is no longer an issue, @elettrico can you confirm?
